gnomad_qc/v4/annotations/generate_freq.py

```diff
@@ -110,37 +109,56 @@
 
 
 def get_freq_resources(
-    overwrite: bool = False, test: Optional[bool] = False, chrom: Optional[str] = None
+    overwrite: bool = False,
+    test: Optional[bool] = False,
+    chrom: Optional[str] = None,
+    ukb: bool = False,
+    non_ukb: bool = False,
 ) -> PipelineResourceCollection:
     """
     Get frequency resources.
 
     :param overwrite: Whether to overwrite existing files.
     :param test: Whether to use test resources.
     :param chrom: Chromosome used in freq calculations.
+    :param ukb: Whether to get frequency resources for UKB subset.
```

Comment:
basic question: do you need to create a UKB subset to make the frequency calculations complete? (asking because I thought the only v4 subset was non-ukb)

Reply:
yeah, they are handled differently. The non-ukb subset actually makes a subset, they other one only makes the standard freqs and then they are added together. This subsetting is just for parallelization and smaller densify purposes